pkg/kube-metrics/metrics.go

```diff
@@ -111,3 +122,22 @@ func GetNamespacesForMetrics(operatorNs string) ([]string, error) {
 	}
 	return ns, nil
 }
+
+func isNamespaced(gvk schema.GroupVersionKind, cfg *rest.Config) (bool, error) {
+	discoveryClient, err := discovery.NewDiscoveryClientForConfig(cfg)
+	if err != nil {
+		log.Error(err, "Unable to get discvery client")
```

Comment:
```suggestion
		log.Error(err, "unable to get discvery client")
```

Reply:
it should solve the issue faced in the sanity

Reply:
done